### DISCUSSION TITLE
NANO 33 IOT Support

### DISCUSSION THREAD

**Discussion by SpamWall** [feilipu/Arduino_FreeRTOS_Library#62] [Q&A]
The new NANO 33 IOT uses the SAM D21 family chip and there is an error compiling Arduino_FreeRTOS - it can't find <avr/wdt.h>.  Is there work being done to support this chip?

**Comment by feilipu** [comment]
See issue #47. SAM is not AVR architecture.

Hence will not be supported by this library. Sorry.
